google/cloud/forseti/scanner/scanners/ke_jmespath_scanner.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+# Copyright 2018 Ross Vandegrift. All rights reserved.
```

Comment:
We should not have individual copyrights.  Instead, can you please add your organization in the `AUTHORS` file?
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/AUTHORS

Can you please fix everywhere?

Reply:
Yes - pushing a fix shortly.